scripts/pr-convergence.py

```
@@ -0,0 +1,1186 @@
+                "-F",
+                f"owner={self.owner}",
+                "-F",
+                f"name={self.name}",
+                "-F",
+                f"headRefName={head_ref_name}",
+            ]
+            if after:
+                args.extend(["-F", f"after={after}"])
+            data = self.run(args)
+            if not isinstance(data, dict):
+                raise GhReadError("GraphQL response was not an object")
```

Ruff (0.16.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)